package.json

```diff
@@ -68,7 +68,7 @@
         "ts-jest": "^26.4.2"
     },
     "scripts": {
-        "build:all": "nx build @deriv/components --skip-nx-cache && nx run-many --target=build",
+        "build:all": "nx build @deriv/components --skip-nx-cache && nx build @deriv/account --skip-nx-cache && nx run-many --target=build",
```

Comment:
Do we need this all the time or can we remove it after the build?

Reply:
all of the builds should happen in this order.

Reply:
As far as I remember the build process was killed for other packages as well and not just the account. But we can test this out to see if it works.

Reply:
running the circleci runner to see if it's working or not

Reply:
https://app.circleci.com/pipelines/github/ali-hosseini-deriv/deriv-app?branch=ako%2FFEQ-400%2Ffix-CI-failings
Working fine it seems